Jenkinsfile

```diff
@@ -48,7 +48,7 @@ pipeline {
             docker.image("selenium/standalone-firefox:4").withRun("-e START_XVFB=false --shm-size=2g --name ${seleniumName} --network ${networkName}") {
               docker.build('maven').inside("--name ${ivyName} --network ${networkName}") {
                 maven cmd: 'clean verify ' +
-                      "-Divy.engine.version='[10.0.0,]' " +
+                      "-Divy.engine.version='[10.0.0, 10.1.0]' " +
```

Comment:
hi @alexsuter . this is correct right?

Reply:
I think it should be `[10.0.0, 10.1.0) `

Reply:
I think [ ] should be also correct?
![image](https://user-images.githubusercontent.com/45038833/197170322-b9bcc5c3-1b4b-4e4f-9e71-5da90b055fd8.png)

Reply:
So 10.1.0 is ok but 10.1.1 not?

Reply:
solution of cst is more correct. you should only search for 10.0.x

but we anyway will never release 10.1. I guess so.

rather than tweaking it here, we should improve the project build plugin, so that this is no longer possible at all